src/app/features/components/scroll-top/scroll-top.component.scss

```diff
@@ -0,0 +1,13 @@
+
+.scroll-to-top {
```

Comment:
put all he css things insie :host tag, is like this

:host {
evering here
}

This is because it means that this css classes are only for these components, if not, you are applying the css classes to the all application and it can bring some issues in the future